consensus/poet/families/sawtooth_validator_registry/validator_reg_message_factory.py

```diff
@@ -287,3 +289,21 @@ def create_set_request_validator_map(self):
     def create_set_response_validator_map(self):
         addresses = [self._key_to_address("validator_map")]
         return self._factory.create_set_response(addresses)
+
+    def create_get_request_report_key_pem(self):
+        return \
+            self._factory.create_get_request(
+                ['000000a87cb5eafdcca6a87ccc804f5546a'
```

Comment:
Yes, this is only meant to be used in testing. It needed a lot of it to be hardcoded to be able to run the tp tests. I think it would make sense to move it into the tests directory.